types/msg.go

```diff
@@ -255,6 +257,26 @@ type CloseChannelMsg struct {
 	ChannelID string `json:"channel_id"`
 }
 
+type PayPacketFeeMsg struct {
+	Fee IBCFee `json:"fee"`
+	// Allowlist of relayer addresses that can receive the fee. This is currently not implemented and *must* be empty.
+	Relayers Array[string] `json:"relayers"`
+	Src      IBCEndpoint   `json:"src"`
+}
+type PayPacketFeeAsyncMsg struct {
+	Fee IBCFee `json:"fee"`
+	// Allowlist of relayer addresses that can receive the fee. This is currently not implemented and *must* be empty.
+	Relayers Array[string] `json:"relayers"`
+	Sequence uint64        `json:"sequence"`
+	Src      IBCEndpoint   `json:"src"`
+}
+
+type IBCFee struct {
+	AckFee     Array[Coin] `json:"ack_fee"`
+	RecvFee    Array[Coin] `json:"recv_fee"`
+	TimeoutFee Array[Coin] `json:"timeout_fee"`
+}
```

Comment:
Could you update this PR to the latest version we merged?

Reply:
Yup, already on my todo list for today